src/AddItem.js

```diff
@@ -3,28 +3,73 @@ import firebase from 'firebase';
 
 const AddItem = () => {
   const [inputValue, setInputValue] = useState();
+  const [frequency, setFrequency] = useState(7);
+  const [success, setSuccess] = useState(false);
 
   const addItem = event => {
     event.preventDefault();
+
     return firebase
       .firestore()
       .collection('items')
-      .add({ name: inputValue });
+      .add({
+        token: '143',
+        name: inputValue,
+        frequency: frequency,
+        lastPurchased: null,
+      })
+      .then(setSuccess(true));
+  };
+
+  const handleOnChange = e => {
+    setFrequency(parseInt(e.target.value));
   };
 
   return (
     <div>
       <form>
+        <label htmlFor="item">Item</label>
         <input
+          id="item"
           type="text"
           name="items"
           placeholder="enter grocery item here"
           onChange={event => setInputValue(event.target.value)}
         />
+        <fieldset>
+          <label htmlFor="soon">Soon</label>
+          <input
+            id="soon"
+            name="frequency"
+            type="radio"
+            value={7}
+            onChange={handleOnChange}
+            checked={frequency === 7}
```

Comment:
You can also use "defaultChecked" here and remove checked properties on two other radio buttons :)